src/assets/AssetsController.ts

```diff
@@ -190,24 +189,6 @@ export class AssetsController extends BaseController<AssetsConfig, AssetsState>
     this.hub.emit(`${suggestedAssetMeta.id}:finished`, failedSuggestedAssetMeta);
   }
 
-  /**
-   * Get collectible tokenURI API following ERC721
-   *
-   * @param contractAddress - ERC721 asset contract address
-   * @param tokenId - ERC721 asset identifier
-   * @returns - Collectible tokenURI
-   */
-  private async getCollectibleTokenURI(contractAddress: string, tokenId: number): Promise<string> {
```

Comment:
This should be split into a separate PR. I found that this is the only place that `assetsContract.getCollectibleTokenURI` was called, so the prior check to ensure it supports metadata has been built into `assetsContract.getCollectibleTokenURI` to simplify this interaction.

Reply:
Done in #414